manifest.xml

```diff
@@ -2,7 +2,7 @@
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
 	<name>Account Removal</name>
 	<author>ColdTrick IT Solutions</author>
-	<version>3.0</version>
+	<version>3.1</version>
```

Comment:
Please don't touch version number as this is handled upon release of the plugin

PS: maybe for dev purpose make it 4.0-beta

Reply:
updated